integrations/jenkins/Jenkinsfile

```diff
@@ -130,6 +130,12 @@ pipeline {
             description: 'An optional existing analyzer result file to use instead of running the analyzer.'
         )
 
+        string(
+            name: 'ANALYZER_INPUT_PATH',
+            description: 'The path to analyze, relative to the repository root.',
```

Comment:
You could add here that it is possible to set the path to a definition file to analyze only that single project.